### ISSUE TITLE
Framework does not request all the fields in all the queries

### ISSUE THREAD

**Issue by maxp-hover** [EasyGraphQL/easygraphql-load-tester#51]
So, this took me a bit of time to dig into, and I doubt it's intended behavior (if it is, please tell me). 

Say I have this query defined in my schema:

```
jobs(
   // Args ommited here
): JobPaginatedResults!
```

And we have the following type:

```
type JobPaginatedResults {
  pagination: Pagination
  results: [Job]
}
```

From my understanding, this library should be requesting _all_ those fields (everything inside Pagination, and in Job), however it's doing something different. 

It's actually building _two distinct queries_ - the first one _only_ has the Pagination fields, and the second has _both_ Pagination and Job.

I think this is caused by lines 53-60 in index.js:

```
nestedType.fields.forEach((field) => {
  const createdField = getField(field, schema)
  fields.push(createdField)

  const queryToTest = createQueryToTest(fields, queryHeader, isMutation)

  queriesToTest.push(queryToTest)
})
```

So, when it gets to the first iteration, it pushes Pagination into the fields list, and builds a query for just Pagination. The next iteration, it adds Results into the fields list, and builds another query - for both fields.

As a workaround, I have made the following modification:

```
nestedType.fields.forEach((field) => {
  const createdField = getField(field, schema)
  fields.push(createdField)
})

const queryToTest = createQueryToTest(fields, queryHeader, isMutation)

queriesToTest.push(queryToTest)
```

Now it only actually builds one query, using both the fields. This seems like the correct behavior. What do you think? Should I submit a PR? I'm currently using a fork I made. Does my explanation make sense, or is it confusing (Sorry if that's the case, but I can try and explain better)?

**Comment by estrada9166** [comment]
This may be fixed in the [`v1.0.0-beta.2`](https://www.npmjs.com/package/easygraphql-load-tester/v/1.0.0-beta.2)
If you can give it a try it'll be awesome!

**Comment by estrada9166** [comment]
This should be fixed in [`v2.0.1`](https://www.npmjs.com/package/easygraphql-load-tester/v/2.0.1)
